analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AssertionArgsShouldBePassedInCorrectOrder.Xunit.cs

```diff
@@ -27,3 +27,20 @@ public void Foo()
         }
     }
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/6630
+namespace Repro_6630
+{
+    class Program
+    {
+        [Fact]
+        public void Foo()
+        {
+            var str = "";
+            Assert.Equal(actual: "", expected: str); // Compliant FN
+            Assert.Equal(expected: "", actual: str); // Compliant
+            Assert.Equal(actual: str, expected: ""); // Noncompliant FP
+            Assert.Equal(expected: str, actual: ""); // Noncompliant
```

Comment:
Do we support all the "Equals" variants described here?
https://xunit.net/docs/comparisons
* Equal
* NotEqual
* NotSame
* Same
for xunit and the corresponding methods for the other frameworks. If so, we should add more test cases (the parameter names might differ) and if not we need another issue.

Reply:
For both **MsTest** and **NUnit** we support:
- `AreEqual`
- `AreNotEqual`
- `AreSame`
- `AreNotSame`
For **XUnit**:
- `Equal`
- `Same`
- `NotSame`

The rule is checking whether MemberAccessExpressionSyntax expression name is one of the above so in order to keep the test cases as minimal as possible I think we can avoid adding those as well, WDYT?

Reply:
You are right indeed. I'll add those test cases as well.